crates/astria-core/src/sequencerblock/v1alpha1/celestia.rs

```diff
@@ -435,7 +438,7 @@ impl UncheckedCelestiaSequencerBlob {
         }?;
         let rollup_ids: Vec<_> = rollup_ids
             .into_iter()
-            .map(RollupId::try_from_vec)
+            .map(|raw| RollupId::try_from_raw(&raw))
```

Comment:
Very minor comment:

You could express this more elegantly (similar to before) like so:
```rust
let rollup_ids: Vec<_> = rollup_ids
    .iter()
    .map(RollupId::try_from_raw)
```